dds_web/crypt/key_gen.py

```diff
@@ -0,0 +1,74 @@
+""" Code for generating project related keys """
+
+from os import urandom
+
+from cryptography.hazmat import backends
+from cryptography.hazmat.primitives import serialization
+from cryptography.hazmat.primitives.asymmetric import x25519
+from cryptography.hazmat.primitives.kdf import scrypt
+from nacl import bindings
+from flask import current_app
+
+
+class project_keygen(object):
```

Comment:
I would like to rename this a bit
* `project_keygen` --> `ProjectKeys`
* `get_key_info_dict` --> `key_dict`
* `_set_project_id_bytes` --> `_projectid_to_bytes`
* `_set_salt_and_nonce` --> `_gen_salt_and_nonce`
* `_set_private_and_public_keys` --> `_generate_keypair`

Reply:
Done!